config/components/webhook/manifests.yaml

```
@@ -4,6 +4,25 @@ kind: MutatingWebhookConfiguration
 metadata:
   name: mutating-webhook-configuration
 webhooks:
+- admissionReviewVersions:
```

Is this change required, or just formatting? If formatting, then I would suggest to revert.

---

I believe it's automatically added on `make verify`.

---

Hm, but it is only moved from another place. Why wasn't this then moved prior to this PR?

---

I think it's because `_ "sigs.k8s.io/kueue/pkg/controller/jobs/deployment"` added  on `pkg/controller/jobs/jobs.go` and sort list changed on code-generator. In any case, `make verify` should throw an error if something not correct.

---

sgtm, thanks!